pyanaconda/installation.py

```diff
@@ -100,15 +100,17 @@ def _prepare_configuration(storage, payload, ksdata):
     services_dbus_tasks = services_proxy.InstallWithTasks()
     os_config.append_dbus_tasks(SERVICES, services_dbus_tasks)
 
-    os_config.append(Task("Configure keyboard", ksdata.keyboard.execute))
-
     # add installation tasks for the Timezone DBus module
     timezone_proxy = TIMEZONE.get_proxy()
     timezone_dbus_tasks = timezone_proxy.InstallWithTasks()
     os_config.append_dbus_tasks(TIMEZONE, timezone_dbus_tasks)
 
-    # add installation tasks for the Localization DBus module
+    # get missing keyboard values
     localization_proxy = LOCALIZATION.get_proxy()
+    convert_task = localization_proxy.ConvertMissingKeyboardConfigurationWithTask()
```

Comment:
I would include this task in `InstallWithTasks` if possible.

Reply:
Modified to PopulateMissing... task. It is used for example when nothing is set (by ks or boot options) and x_layout is inferred from lang. In this case the task updates vc_keymap. I think we could remove this task when looking at updating of the values from lang. One thing to consider is that this update seems to happen only from GUI. It is for another PR.